src/pyquil/quilbase.py

```diff
@@ -2595,70 +2504,71 @@ def __deepcopy__(self, memo: Dict) -> "DefCircuit":
         return DefCircuit._from_rs_circuit_definition(super().__deepcopy__(memo))
 
 
-class DefCalibration(quil_rs.Calibration, AbstractInstruction):
-    def __new__(
-        cls,
-        name: str,
-        parameters: Sequence[ParameterDesignator],
-        qubits: Sequence[Union[Qubit, FormalArgument]],
-        instrs: Sequence[AbstractInstruction],
-        modifiers: Optional[List[quil_rs.GateModifier]] = None,
-    ) -> Self:
-        return super().__new__(
-            cls,
-            name,
-            _convert_to_rs_expressions(parameters),
-            _convert_to_rs_qubits(qubits),
-            _convert_to_rs_instructions(instrs),
-            modifiers or [],
-        )
-
-    @classmethod
-    def _from_rs_calibration(cls, calibration: quil_rs.Calibration) -> "DefCalibration":
-        return super().__new__(
-            cls,
-            calibration.name,
-            calibration.parameters,
-            calibration.qubits,
-            calibration.instructions,
-            calibration.modifiers,
-        )
-
-    @property  # type: ignore[override]
-    def parameters(self) -> Sequence[ParameterDesignator]:
-        return _convert_to_py_expressions(super().parameters)
-
-    @parameters.setter
-    def parameters(self, parameters: Sequence[ParameterDesignator]) -> None:
-        quil_rs.Calibration.parameters.__set__(self, _convert_to_rs_expressions(parameters))  # type: ignore[attr-defined] # noqa
-
-    @property  # type: ignore[override]
-    def qubits(self) -> List[QubitDesignator]:
-        return _convert_to_py_qubits(super().qubits)
-
-    @qubits.setter
-    def qubits(self, qubits: Sequence[QubitDesignator]) -> None:
-        quil_rs.Calibration.qubits.__set__(self, _convert_to_rs_qubits(qubits))  # type: ignore[attr-defined]
-
-    @property
-    def instrs(self) -> List[AbstractInstruction]:
-        return _convert_to_py_instructions(super().instructions)
-
-    @instrs.setter
-    def instrs(self, instrs: Sequence[AbstractInstruction]) -> None:
-        quil_rs.Calibration.instructions.__set__(self, _convert_to_rs_instructions(instrs))  # type: ignore[attr-defined] # noqa
-
-    def out(self) -> str:
-        return super().to_quil()
-
-    def __str__(self) -> str:
-        return super().to_quil_or_debug()
-
-    def __copy__(self) -> Self:
-        return self
-
-    def __deepcopy__(self, memo: Dict) -> "DefCalibration":
-        return DefCalibration._from_rs_calibration(super().__deepcopy__(memo))
+# class DefCalibration(quil_rs.Calibration, AbstractInstruction):
```

Comment:
Delete

Reply:
Ah, I was toggling between using this, and not, for benchmarking and testing compatibility. I've re-enabled it, since the intent of this state is to be able to test both implementations, if desired.